pages/home.page.php

```diff
@@ -1,3 +1,6 @@
+<head>
+    <meta http-equiv="refresh" content="300;url=https://pokemap-bergstrasse.de/">
```

Comment:
Please noooooo 😨 There is aru.php for refresh. I don't get this? + custom url...